webpages/StaffPage.php

```diff
@@ -84,4 +85,22 @@
 </div>
 </div>
 
+<div class="row">
+<div class="col-lg-8">
+<div class="card mt-2">
```

Comment:
Should this be visible to everyone? Or just site admins?

Reply:
I mention this in #163.

"Everyone" in this context means everyone with access to the staff overview page. For conventions I'm involved with, that tends to be a fairly small group, and I'd prefer to have the programme team seeing this so they can nag the admins to fix it.

However, other conventions might want to restrict what users can see it.

I could add a permission to view this pane. I would be inclined to have it in the default permissions for admin and staff, and conventions that don't want staff seeing this can delete it from the staff role.

Reply:
Okay, I can accept that.

Reply:
I'll open an issue to discuss adding a permission to see "Site status" panel. I'm reluctant to put work into it unless people are looking for it.